### PR TITLE
SAK-42175 Standardardized Banners in Site-manage and Assignment

### PR DESCRIPTION
**Pull request by michelle-tai**
https://jira.sakaiproject.org/browse/SAK-42175

Standardardized Banners in Site-manage and Assignment according to https://jira.sakaiproject.org/browse/SAK-41866

### CODE REVIEW

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_delete_assignment.vm [hunk 1]
```diff
@@ -9,11 +9,11 @@
 	</h3>
 
 		#if ($alertMessage)
-			<div class="alertMessage">  <strong>$tlang.getString("gen.alert")</strong>  $alertMessage</div><div class="clear"></div>
+			<div class="sak-banner-error">  <strong>$tlang.getString("gen.alert")</strong>  $alertMessage</div>
 		#else
-			<div class="alertMessage">
+			<div class="sak-banner-error">
```
Comment (fostersdesign) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_instructor_delete_assignment.vm]: Should this be .sak-banner-warn for the confirmation?

File: assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm [hunk 2]
```diff
@@ -8,8 +8,8 @@
 
     #set($allowUpdateAssignment = $!service.allowUpdateAssignment($assignmentReference))
     #if ($alertMessage)
-        <div class="alertMessage"><strong>$tlang.getString("gen.alert")</strong> $alertMessage</div>
-        <div class="clear"></div>#end
+        <div class="sak-banner-error"><strong>$tlang.getString("gen.alert")</strong> $alertMessage</div>
+        #end
```
Comment (fostersdesign) [inline assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm]: This `#end` should be aligned with the `#if` please.

File: site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html [hunk 3]
```diff
@@ -66,12 +66,12 @@
...
 <div class="portletBody container-fluid">
 	<div rsf:id="error-row:">
-		<div class="alertMessage" rsf:id="error">We Have Errors!</div>
+		<div class="sak-banner-error" rsf:id="error">We Have Errors!</div>
 	</div>
 
 	<form rsf:id="groups-form" id="groups-form">
 		<h3 rsf:id="prompt"></h3>
-		<p id="" class="messageValidation" rsf:id="emptyGroupTitleAlert" style="display:none"/>
+		<p id="" class="sak-banner-success" rsf:id="emptyGroupTitleAlert" style="display:none"/>
```
Comment (fostersdesign) [inline site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html]: Should this be a warning message?

File: site-manage/site-manage-group-helper/tool/src/webapp/content/templates/GroupEdit.html [hunk 4]
```diff
@@ -46,11 +46,11 @@
 <body>
 <div class="portletBody">
 	<div rsf:id="error-row:">
-		<div class="alertMessage" rsf:id="error">We Have Errors!</div>
+		<div class="sak-banner-error" rsf:id="error">We Have Errors!</div>
 	</div>
 	<form rsf:id="groups-form" onsubmit="return checkform(this);">
 		<h3 rsf:id="prompt"></h3>
-		<p id="" class="messageValidation" rsf:id="emptyGroupTitleAlert" style="display:none"/>
+		<p id="" class="sak-banner-success" rsf:id="emptyGroupTitleAlert" style="display:none"/>
```
Comment (fostersdesign) [inline site-manage/site-manage-group-helper/tool/src/webapp/content/templates/GroupEdit.html]: Should this be a warning message?

**Comment by fostersdesign** [comment]
@michelle-tai, there are some merge conflicts with some of the files that you've changed in this PR. They have changed since you submitted. Can you please update your master branch and then rebase your local branch for this PR and push that again? Thanks.

File: site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html [hunk 5]
```diff
@@ -66,14 +66,12 @@
...
-		<div class="alertMessage" rsf:id="error">We Have Errors!</div>
+		<div class="sak-banner-error" rsf:id="error">We Have Errors!</div>
 	</div>
 
 	<form rsf:id="groups-form" id="groups-form">
-		<div class="page-header">
-			<h1 rsf:id="prompt"></h1>
-		</div>
-		<p id="" class="messageValidation" rsf:id="emptyGroupTitleAlert" style="display:none"/>
+		<h3 rsf:id="prompt"></h3>
```
Comment (fostersdesign) [inline site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html]: I think you've removed the new page-header format. Was this on purpose?

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-newSiteCourse.vm [hunk 6]
```diff
@@ -1,18 +1,15 @@
...
-			<h1>$tlang.getString("nscourse.creating")</h1>
-			<p class="step">
-				$tlang.getString("nscourse.course_sections_selection") - $term.title
-			</p>
-		#else
-			<h1>$tlang.getString("nscourse.edit") <span class="highlight">$!siteTitle</span></h1>
-		#end
-	</div>
-	#if ($alertMessage)<div class="alertMessage">$tlang.getString("gen.alert") $alertMessage</div>#end
+	#if (!$!site)
```
Comment (fostersdesign) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-newSiteCourse.vm]: Another case of the page heading being reverted...

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm [hunk 7]
```diff
@@ -3,10 +3,10 @@
 
         <input type="hidden" name="eventSubmit_doManage_overview_option" value="x" />
         <input type="hidden" name="option" value="cancel" />
-        <div class="page-header">
-            <h1>$tlang.getString("manover.header") <span class="highlight">$!site.Title</span></h1>
-        </div>
-        #if ($alertMessage)<div class="alertMessage">$tlang.getString("manover.alert") $validator.escapeHtml($alertMessage)</div>#end
+        <h3>
```
Comment (fostersdesign) [inline site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm]: Another case of the page heading being reverted...

Comment (michelle-tai) [reply site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm]: Is this better? I think a reason it was being reverted was that in master, there's places where there's conflicts (like some alertmessage areas) which affected what ended up being pushed

Comment (fostersdesign) [reply site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm]: No, it is still showing the removal of the .page-header div and the change back to an h3 here in GitHub.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-newSiteCourse.vm (hunk 6 above)
Comment (fostersdesign) [reply site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-newSiteCourse.vm]: This .page-header is still being reverted.

File: site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html (hunk 5 above)
Comment (fostersdesign) [reply site-manage/site-manage-group-section-role-helper/tool/src/webapp/content/templates/GroupEdit.html]: This .page-header has still been reverted.

File: site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm (hunk 7 above)
Comment (fostersdesign) [reply site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-siteInfo-manageOverview.vm]: Oh, sorry: it took me to the old version. Let me do another review...

File: site-manage/site-manage-link-helper/src/webapp/vm/sakai_link.vm [hunk 8]
```diff
@@ -4,7 +4,7 @@
         <h1>$tlang.getString('title')</h1>
     </div>
 #if ($alertMessage)
-    <div class="alertMessage">$tlang.getString("gen.alert") $alertMessage</div><div style="display:block;clear:both" ></div>
+    <div class="sak-banner-error">$tlang.getString("gen.alert") $alertMessage</div><div style="display:block;clear:both" ></div>
```
Comment (fostersdesign) [inline site-manage/site-manage-link-helper/src/webapp/vm/sakai_link.vm]: Missed removing a clear block div.

File: site-manage/site-manage-link-helper/src/webapp/vm/sakai_link.vm [hunk 9]
```diff
@@ -33,7 +33,7 @@
 </select>
 #else
 <p>
-<div class="alertMessage">$tlang.getString('error.nosites')</div><div style="display:block;clear:both" ></div>
+<div class="sak-banner-error">$tlang.getString('error.nosites')</div></div>
```
Comment (fostersdesign) [inline site-manage/site-manage-link-helper/src/webapp/vm/sakai_link.vm]: Extra closing div tag left over from removing the clear block div.

**Comment by michelle-tai** [comment]
I made the changes–let me know if there's anything I missed! @fostersdesign

**Pull request merged**